FWCore/Framework/test/test_catch2notTP_MergeableRunProductMetadata.cc

```diff
@@ -168,8 +168,8 @@ TEST_CASE("test MergeableRunProductMetadata", "[MergeableRunProductMetadata]") {
 
     REQUIRE(mergeableRunProductProcesses.size() == 2);
 
-    std::vector<std::string> expected{"AAPROD", "APROD"};
-    REQUIRE(mergeableRunProductProcesses.processesWithMergeableRunProducts() == expected);
+    std::vector<std::string> expectedProcesses{"AAPROD", "APROD"};
```

Comment:
Try moving these two lines into their own scope to limit lifetime of `expectedProcesses`

Reply:
Done